pkg/dss/scd/cockroach/operations.go

```diff
@@ -270,6 +278,10 @@ func (s *Store) DeleteOperation(ctx context.Context, id scdmodels.ID, owner dssm
 		return nil, nil, multierr.Combine(err, tx.Rollback())
 	}
 
+	if _, err := tx.ExecContext(ctx, deleteImplicitSubscriptionQuery, old.SubscriptionID, owner); err != nil {
```

Comment:
I think we can delete the implicit Subscription before searching for Subscriptions to return because the caller shouldn't have to notify themself that they deleted their Operation in response to deleting their Operation.

Reply:
I have been pondering on that a little bit. From a client's perspective, it could very well be easier (as in "more uniformly") to just notify itself of the deleted operation. I don't have a strong opinion, but not returning a client's own subscription seems like special-casing without the requirement to do so.

Reply:
I don't think I have a strong opinion, but my thought here is that at the time the response is returned to the client, the Subscription no longer exists.  I could buy that its last duty was to trigger this notification, but it seems not very useful to notify someone of something when they only want to know about things relevant to the Operation that no longer exists

Reply:
ACK, let's keep it like it is right now and fix *if* we find problems. Right now, executing it here means less re-shuffling (somewhat arbitrary objective).